src/shared/dtos/pagination-params.dto.ts

```diff
@@ -9,6 +9,7 @@ export class PaginationParamsDto {
   })
   @IsNumber()
   @IsOptional()
+  @Min(1)
```

Comment:
@Seb-C 
What if I just want to get the `totalCount` of some resources, which is returned in the `meta` property, without fetching any actual resources by setting the limit as `0`? :thinking:

Reply:
@yashmurty LGTM! https://github.com/monstar-lab-oss/nestjs-starter-rest-api/pull/207/commits/300400bfb60826d603cd1e88be2ce804a52e856f